### PR TITLE
Fixing STB image symbol duplicates 

### PR DESCRIPTION
**Pull request by dcoeurjo**
# PR Description

Fixes stb image symbol duplicates (#1714) following @nzfeng's suggestions.

# Checklist

- [x] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Documentation module page added or updated.
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtal/blob/master/ChangeLog.md) added.
- [x] No warning raised in Debug mode.
- [x] All continuous integration tests pass (Github Actions)

### CODE REVIEW

**Comment by dcoeurjo** [comment]
@kerautret could you please have a look to this one? It seems to be related to issues you've fixed for DGtalTools.

**Comment by kerautret** [comment]
> @kerautret could you please have a look to this one? It seems to be related to issues you've fixed for DGtalTools.

Yes thanks I look it, perhaps it could avoid the fix inside the tools ;)

File: src/DGtal/math/linalg/EigenDecomposition.ih [hunk 1]
```diff
@@ -215,10 +215,8 @@ decomposeQL( Matrix& V, Vector& d, Vector e )
       // otherwise, iterate.
       if( m > l )
         {
-          int iter = 0;
```
Comment (kerautret) [inline src/DGtal/math/linalg/EigenDecomposition.ih:218]: not related but clean looks fine ;)

**Review by kerautret — APPROVED** [review]
Looks fine, since the tools issue was only on windows, I think that we could merge and I will check if the fix can be remove (else quite non direct to check on CI with multiple branches) ;)

File: src/DGtal/math/linalg/EigenDecomposition.ih (hunk 1 above)
Comment (dcoeurjo) [reply src/DGtal/math/linalg/EigenDecomposition.ih:218]: yeah I know.. my bad.. I saw this warning and couldn't resist to fix it ;)

**Comment by dcoeurjo** [comment]
thx

**Comment by dcoeurjo** [comment]
Merging.

**Pull request merged**